Tests/BenchmarkTests/BenchmarkMetricsTests.swift

```diff
@@ -61,7 +64,10 @@ final class BenchmarkMetricsTests: XCTestCase {
         "readBytesLogical",
         "writeBytesLogical",
         "readBytesPhysical",
-        "writeBytesPhysical"
+        "writeBytesPhysical",
+        "retainCount",
+        "releaseCount",
+        "retainReleaseDelta"
```

Comment:
```suggestion
        "retainReleaseDelta",
```

to avoid extra changes on further additions to this array?

Reply:
We have this lint rule currently:

/Users/hassila/GitHub/package-benchmark/Tests/BenchmarkTests/BenchmarkMetricsTests.swift:70:29: warning: Trailing Comma Violation: Collection literals should not have trailing commas. (trailing_comma)

Maybe should consider changing it....